Primary colors must be opaque

It looks like we've also had some issues with this, see [here](https://stackoverflow.com/a/29166908)

Here's a log of it

```java
FATAL EXCEPTION: main
Process: org.hackillinois.androidapp, PID: 16488
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.hackillinois.androidapp/org.hackillinois.android.ui.SplashActivity}: java.lang.RuntimeException: A TaskDescription's primary color should be opaque
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2406)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2466)
	at android.app.ActivityThread.access$1200(ActivityThread.java:152)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1341)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5538)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
Caused by: java.lang.RuntimeException: A TaskDescription's primary color should be opaque
	at android.app.ActivityManager$TaskDescription.<init>(ActivityManager.java:536)
	at android.app.Activity.onApplyThemeResource(Activity.java:3735)
	at android.view.ContextThemeWrapper.initializeTheme(ContextThemeWrapper.java:140)
	at android.view.ContextThemeWrapper.setTheme(ContextThemeWrapper.java:85)
	at android.support.v7.app.e.setTheme(Unknown Source)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2349)
	... 10 more
```

--- a/app/release/output.json
+++ b/app/release/output.json
@@ -1,1 +1,1 @@
-[{"outputType":{"type":"APK"},"apkInfo":{"type":"MAIN","splits":[],"versionCode":11},"path":"hackillinois-v1.1.6.apk","properties":{"packageId":"org.hackillinois.androidapp","split":"","minSdkVersion":"19"}}]
+[{"outputType":{"type":"APK"},"apkInfo":{"type":"MAIN","splits":[],"versionCode":12},"path":"hackillinois-v1.1.7.apk","properties":{"packageId":"org.hackillinois.androidapp","split":"","minSdkVersion":"19"}}]

--- a/app/src/main/res/values/colors.xml
+++ b/app/src/main/res/values/colors.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="colorPrimary">@color/colorPrimaryDark</color>
-    <color name="colorPrimaryDark">#801C055E</color>
+    <color name="colorPrimaryDark">#514186</color>
     <color name="pink">#ED008C</color>
 
     <color name="mainBackground">#F2F5Fe</color>

